spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -318,4 +326,31 @@ public AutoTimeProperties getAutotime() {
 
 	}
 
+	public static class Jvm {
```

Comment:
So far all instrumented metrics seem to have their config props in here. It does feel like it is starting to get a bit overloaded, especially w/ this addition. My gut instinct is to create a separate `JvmMetricsConfigurationProperties` (still prefixed at `management.metrics.jvm`) but I was unsure if it is intentional to have all things metrics config props to be in here.